OptionExt/ValueOption.fs

```diff
@@ -0,0 +1,34 @@
+module OptionExt.ValueOption
```

Comment:
we should add converters to/from Option in here.  (And converters to/from ValueOption in the Option module)

Reply:
✔️